workflow/README.md

```diff
@@ -43,17 +43,29 @@ For both short and long reads. Different parameters are recommended for each seq
 - Supported aligners: [minimap2](https://github.com/lh3/minimap2), [winnowmap2](https://github.com/marbl/Winnowmap),
 [bwa mem](https://github.com/lh3/bwa) and [Bowtie 2](http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml) 
 
-Minimap2 example:
+Minimap2 example with Pacbio:
 ```
 bash leviosam2.sh \
     -a minimap2 -g 1000 -H 100 -S -x ../configs/pacbio_all.yaml \
+    -l map-hifi \
     -i pacbio.bam \
     -o pacbio-lifted \
     -f grch38.fna \
     -C chm13v2-grch38.clft \
     -t 16
 ```
 
+Minimap2 example with Nanopore:
+```
+bash leviosam2.sh \
+    -a minimap2 -g 1000 -H 100 -S -x ../configs/ont_all.yaml \
```

Comment:
As I also tried using ONT in testing, I want to leave comments here for your reference.
1. the gap and edit distance settings are better configured differently to hifi, as ONT read can be very long and far more gaps.
```
# i tested these configs have better performance although having edit distance cutoff is not a good idea as long read have very diverse read length compared to Illumina read which have similar length.
-a minimap2 -g 5000 -H 1000
```
2. the ont_all.yaml is configured as below in my testing, it is the same as in minimap2 default setting for ONT I think, maybe you can also share the setting for the ONT support.
````
alignment:
    engine: "ksw_extd2_sse"
    nm_threshold: 0
    flag: 0x08 # KSW_EZ_APPROX_MAX
    a: 2
    b: 4
    q: 4
    e: 2
    q2: 24
    e2: 1
    w: 601 # w = bw * 1.5 + 1
    zdrop: 400
    end_bonus: -1
```
Hope this can help,
JH

Reply:
Thanks for the reply @sujunhao! I haven't done enough testing with this to know what the best numbers are. I'll let the authors decide.

Reply:
@zhemingfan @sujunhao Many thanks for the inputs! I'm on a trip and will test the PR and parameters when I get back.